EventKit/eventkit-cloud
# Update Webpack

## Description of Improvement

Update webpack and related packages, update webpack config.

## How to test

Pull down the changes and bring down your webpack container.  Rebuild webpack using `docker-compose build --no-cache webpack`.  Bring your webpack container back online.  Ensure that it builds properly and that you can access the application without any issues.

## Quality Assurance 

The following items have been updated:
 
## Screenshots

:smile:
